source/default-conf.py

```diff
@@ -173,6 +173,32 @@
         'reference/kubectl-minio-plugin/kubectl-minio-tenant.rst',
         'reference/kubectl-minio-plugin/kubectl-minio-version.rst',
     ]
+elif tags.has("container"):
+    excludes = [
+        'operations/install-deploy-manage/deploy-minio-tenant.rst',
+        'operations/install-deploy-manage/modify-minio-tenant.rst',
+        'operations/install-deploy-manage/expand-minio-tenant.rst',
+        'operations/install-deploy-manage/upgrade-minio-tenant.rst',
+        'operations/install-deploy-manage/upgrade-minio-operator.rst',
+        'operations/install-deploy-manage/delete-minio-tenant.rst',
+        'operations/install-deploy-manage/minio-operator-console.rst',
+        'operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.rst',
+        'operations/install-deploy-manage/multi-site-replication.rst',
```

Comment:
Intentionally excluded in this pass.

- Distributed deploys via containers should be done via K8s. We will not be publishing docs for compose, swarm, helm, etc.
- Multi-Site can be handled in a separate pass if at all, as container networking makes this very finicky.